syfertext/doc.py

```diff
@@ -4,10 +4,12 @@
 
 hook = syft.TorchHook(torch)
 
+from typing import Dict
 from syft.generic.object import AbstractObject
```

Comment:
Would you please add this import next to the other import of the `typing` library below